chrono/urls.py

```diff
@@ -15,7 +15,10 @@
 """
 from django.contrib import admin
 from django.urls import path
+from django.views.decorators.csrf import csrf_exempt
+from graphene_django.views import GraphQLView
 
 urlpatterns = [
     path('admin/', admin.site.urls),
+    path('graphql/', csrf_exempt(GraphQLView.as_view(graphiql=True))),
```

Comment:
Add an `graphiql` view as well. Eases documentation.

Reply:
Also with that plugin @annshress to generate queries.

Reply:
Yes.

https://github.com/OneGraph/graphiql-explorer
https://pypi.org/project/graphene-graphiql-explorer/

This will come in handy later during the development.